rasterio/plot.py

```diff
@@ -31,8 +31,8 @@ def get_plt():
         raise ImportError(msg)
 
 
-def show(source, with_bounds=True, contour=False, contour_label_kws=None,
-         ax=None, title=None, transform=None, adjust=False, **kwargs):
+def show(source, with_bounds=True, contour=False, contour_label_kws=None, indexes=None,
+         ax=None, title=None, transform=None, clip_percent=None, adjust=False, **kwargs):
```

Comment:
```suggestion
         ax=None, title=None, transform=None, percent_range=None, adjust=False, **kwargs):
```

Reply:
the 'clip_percent' has been changed to 'percent_change'.